_resources/assets/layout.css

```diff
@@ -276,3 +276,8 @@ button#CybotCookiebotDialogBodyButtonDetails {
 .container-md {
     max-width: 1140px;
 }
+
+/* Fix so that turning off outer pre does not impact styling */
```

Comment:
Perhaps not really necessary? There's just a bit of extra padding around syntax highlighting without it, that isn't there on about.sourcegraph.com today because of re-adding the wrapping `pre`.